Add Agora Chat Release Notes (#2466)
AgoraIO/agora_doc_source: en-US/markdown/agora-chat/Understand/agora_chat_release_notes_android.md

This page provides release notes for the Agora Chat Android SDK.

## V1.0.8

v1.0.8 was released on September 30, 2022. 

#### New features

- Supports custom chat room attributes.

> - Adds the `setAreaCode` method to restrict the scope of accessible edge nodes.

Reviewer: 这个是新接口吗？怎么没看到 guide 的更新？

Author: 跟研发确认，不需要在 Guide中体现。

> - Adds the `getJoinedGroupsFromServer` method to allow you to get the number of members in a group and your group role when getting the groups that you have joined.

Reviewer: 这个是不是也涉及到要改 chat_group_android 这篇文档里面关于 getJoinedGroupFromServer 的描述？原先文档里只说了调用这个接口可以获得 group list。

Author: 看了一下 chat_group_android，不需要修改。

> - Adds the `getJoinedGroupsFromServer` method to allow you to get the number of members in a group and your group role when getting the groups that you have joined.

Reviewer: ```suggestion
- Adds the `getJoinedGroupsFromServer` method to allow you to get the number of members and your group role when getting the groups that you join.
```

Author: 1

#### Bugs fixed

> - Failures in getting a great number of messages from the server in few scenarios.

Reviewer: ```suggestion
- Failures in getting a large number of messages from the server in few scenarios.
```

Author: 可以

> - The issue of incorrect data statistics.

Reviewer: ```suggestion
- An issue of incorrect data statistics.
```
